### PR TITLE
Rerender, update, and fix lint

### LINKED ISSUE #4: MNT: The tifffile recipe has some lint :(
This is the friendly conda-forge-admin automated user.

I've ran the conda-smithy linter and found some lint in this feedstock :cry:.

Here is what I have got:

  * The recipe must have a `build/number` section.

Thanks!

### PR DESCRIPTION
**Pull request by ocefpaf**
Closes #4

### CODE REVIEW

**Comment by ocefpaf** [comment]
@jakirkham can you take a quick look and merge?

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,11 +1,17 @@
...
-  fn: tifffile-0.7.0.tar.gz
-  url: https://pypi.python.org/packages/source/t/tifffile/tifffile-0.7.0.tar.gz
-  md5: 38c1de9910a1e6b0faac0b84faf8ef72
+  fn: tifffile-{{ version }}.tar.gz
+  url: https://github.com/blink1073/tifffile/archive/v{{ version }}.tar.gz
+  sha256: 218f9c0640eff4d464c5619b1db57f049e92ff6cd2360d146332577f0ec2f312
+
+build:
+    number: 0
+    script: python setup.py install
```
Comment (jakirkham) [inline recipe/meta.yaml]: It looks like we are using `setuptools`, which I didn't realize.

Not that this is on you, but could we please update this with the canonical `--single-version-externally-managed --record=record.txt` and add `setuptools` explicitly to the build dependencies?

If you're too busy @ocefpaf, I can tack it on in a new PR later.

Comment (ocefpaf) [inline recipe/meta.yaml]: > If you're too busy @ocefpaf, I can tack it on in a new PR later.

No problem. I update it to use setuptools.

Comment (jakirkham) [inline recipe/meta.yaml]: Awesome. Thanks.

**Comment by jakirkham** [comment]
LGTM. We will wait for CI (unless there is anything else I should wait for) and then merge. Thanks @ocefpaf.

**Comment by jakirkham** [comment]
Got some failures on Travis CI, but they have nothing to do with the build. They appear to just be issues downloading binaries from anaconda.org or downloading miniconda. Will restart and see if it now passes.

**Comment by jakirkham** [comment]
Sorry, I really dropped the ball on this. Thanks @ocefpaf. Merging...

**Pull request merged**